### DISCUSSION TITLE
在overleaf上无法编译，编译时间过长。

### DISCUSSION THREAD

**Discussion by G200005** [obster-y/XJTU-thesis#131] [Q&A]
使用的dev版本，在overleaf上编译时间过长，请问怎么解决呢？
![image](https://github.com/user-attachments/assets/506918b6-e193-41f4-98e5-8e5c478c5797)

**Comment by obster-y** [comment]
非常抱歉，这个模板是这样的，Overleaf在一年还是多久之前降低了免费用户的时长上限，我的建议是本地编译或者买个premium，或者自己优化一下编译顺序(我感觉去掉和glossary相关的可能会快一点)
